portfolio/src/main/webapp/index.html

```diff
@@ -38,9 +38,22 @@ <h1> About Me</h1>
 			<p> I'm Sylvia Zhang(张梓愉). Originally from Northeastern China, I'm now a rising
 				junior studying computer science at Carnegie Mellon University.
 			</p>
-      <h1> My Reads</h1>
-      <button onclick="getBookRec()">Get a book recommendation~</button>
-      <div id="book-container"></div>
+
+			<h1> My Favorite Trips</h1>
+			<button onclick="getCityRec()">What's my favorite trip destinations?</button>
+			<div id="city-container"></div>
+		</article>
+	</section>
+	<section>
+		<article>
+			<h2> Comment Section</h2>
+			<form action="/comment" method="POST">
+				<textarea name="text-input">Leave your comment here</textarea>
+				<br/><br/>
+				<input type="submit" />
+      </form>
+				<h1> User Comments </h1>
```

Comment:
The indentation here is excessive, prefer sticking to 4 spaces at max.

Reply:
Hmm, the auto-formatting might have done some weird thing. I'll look into that!